src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -41,7 +41,7 @@ export interface AmalgamationFilingIF {
     identifier: string
   }
   amalgamation: {
-    amalgamationType: AmalgamationTypes
+    type: AmalgamationTypes
     nameRequest: NameRequestFilingIF
     nameTranslations: NameTranslationIF[]
     offices: RegisteredRecordsAddressesIF | object
```

Comment:
Hongjing, I have no idea (yet) what the saved businesses list will look like, but it's the next thing I'm looking into.